### PR TITLE
DP-31033 Formstack upgrade

### PR DESCRIPTION
**Pull request by clairesunstudio**
Fixed:
  - description: Fix form page content type blocking styling from Formstack v4 by removing the query parameter.
    issue: DP-31033

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 99b9e216f992a4dea6edab71f8a3db2a432d5473
</p>

File: changelogs/DP-31033.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Fixed:
+  - description: Fix form page content type blocking styling from Formstack v4 by removing the query parameter.
```
Comment (clairesunstudio) [inline changelogs/DP-31033.yml]:
```suggestion
  - description: Fix form page content type blocking styling from Formstack v4 by removing the query parameter, and removed broken `jsonp` parameter
```

File: docroot/themes/custom/mass_theme/templates/field/field--field-form-embed.html.twig [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
 {% for item in items %}
   {#{{ item.content | raw }}#}
-  <div aria-live="polite" aria-relevant="all" class="form_embed feedback"><script type="text/javascript" src="{{ item.url }}?no_style_strict=1&jsonp"></script><noscript><a href="{{ item.url }}" title="Title">Online Form - Title</a></noscript>
+  <div aria-live="polite" aria-relevant="all" class="form_embed feedback"><script type="text/javascript" src="{{ item.url }}?jsonp"></script><noscript><a href="{{ item.url }}" title="Title">Online Form - Title</a></noscript>
```
Comment (clairesunstudio) [inline docroot/themes/custom/mass_theme/templates/field/field--field-form-embed.html.twig]:
```suggestion
  <div aria-live="polite" aria-relevant="all" class="form_embed feedback"><script type="text/javascript" src="{{ item.url }}"></script><noscript><a href="{{ item.url }}" title="Title">Online Form - Title</a></noscript>
```

File: features/ct_form_page.feature [hunk 3]
```diff
@@ -15,15 +15,15 @@ Feature: Form Page Content type
       | field_form_embed:type  | formstack |
       | field_form_embed:value | "<script type="text/javascript" src="http://example.formstack.com/script?nojquery=1"></script><noscript><a href="http://example.formstack.com/noscript" title="Online Form">Test</a></noscript><div><a href="http://example.com/link" title="Web Form Creator">Web Form Creator</a></div>" |
       | moderation_state       | published |
-    Then I should see a script element with the source "http://example.formstack.com/script?nojquery=1&no_style_strict=1&jsonp=1"
+    Then I should see a script element with the source "http://example.formstack.com/script?nojquery=1&jsonp=1"
```
Comment (clairesunstudio) [inline features/ct_form_page.feature]:
```suggestion
    Then I should see a script element with the source "http://example.formstack.com/script?nojquery=1"
```

**Pull request merged**